test/init_helm_promise_test.go

```diff
@@ -83,14 +78,27 @@ var _ = Describe("init helm-promise", func() {
 			By("generating a workflow file with helm resource configure workflow", func() {
 				pipelines := getPipelines(workingDir)
 				Expect(pipelines).To(HaveLen(1))
-				matchHelmResourceConfigurePipeline(pipelines[0], []corev1.EnvVar{{Name: "CHART_URL", Value: "postgres.io/chart"}})
+				matchHelmResourceConfigurePipeline(pipelines[0], []corev1.EnvVar{
+					{Name: "CHART_URL", Value: "https://helm.github.io/examples"},
+					{Name: "CHART_NAME", Value: "hello-world"}})
+			})
+
+			By("including correct gvk and CRD schema in a api.yaml", func() {
+				apiYAML, err := os.ReadFile(filepath.Join(workingDir, "api.yaml"))
+				Expect(err).NotTo(HaveOccurred())
+				var promiseCRD apiextensionsv1.CustomResourceDefinition
+				Expect(yaml.Unmarshal(apiYAML, &promiseCRD)).To(Succeed())
+				matchCRD(&promiseCRD, "syntasso.io", "v1alpha1", "Database", "database", "databases")
```

Comment:
is it possible to do some more all encompassing test that the CRD is actually valid. Something equivalent to `kubectl create -f crd.yaml --dry-run=server`. probably not, but i do think we might have a small gap about producing actually valid CRDs